_pages/saml.md

```diff
@@ -291,6 +291,8 @@ An example authentication request, with indentation added for readability.
 </div>
 </div>
 
+**Note:** We strongly encourage you to cryptographically sign authentication requests, unless there are technical limitations on your side that prevent this. In addition to providing increased security, this allows for seamless rotation of your application's public certificate in the future. All signatures must use RSA SHA-256.
```

Comment:
is there a specific parameter or XML tag that this shows up in? as a hint for partners to know what to add to their requests?

Reply:
I was trying to put myself in a developer's shoes of "how can I check that I am sending this correctly" but yeah seems like a good backlog item if it's not quick to add here